R/Organism.R

```diff
@@ -878,12 +880,15 @@ setMethod("simBac", "Bac", function(object, arena, j, sublb, bacnum, sec_obj="no
   }
   pos <- arena@orgdat[,c('x','y')]
   if(!dead && !arena@stir && object@speed != 0){
-    if(object@chem == ''){
+    if(is.null(object@chem)){
```

Comment:
object@chem == '' does not support many vector elements

Reply:
The test of `is.null` is not the same as the test of `==""`. I'm not sure if this will not lead to problems.

Reply:
Then what about the  "length(object@chem)  == 0" ? The important point is that we have to change it so that supports many vector elements . How is another topic.....

Reply:
I don't think this needs to be changed. The if statement only checks if chemotaxis is enabled or not, so just the case no chem argument is given in Bac constructor.

Reply:
Ok no problem by me! Anyway if we do not like it afterwards, we can change it accordingly! :) As reviewer you can change my commit....I would suggest